Sources/TSCTestSupport/SwiftTestingHelpers.swift

```diff
@@ -0,0 +1,48 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Testing
+
+import TSCBasic
+
+public func assertFileExists(
```

Comment:
I mean, I'd spell it `expectFileExists()` :P

Reply:
That makes sense. I'll update all function names.